src/main/kube-auth-proxy/kube-auth-proxy-cert.injectable.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import directoryForUserDataInjectable from "../../common/app-paths/directory-for-user-data/directory-for-user-data.injectable";
+import { createKubeAuthProxyCertificateFiles, getKubeAuthProxyCertificatePath } from "./kube-auth-proxy-cert";
+
+const getKubeAuthProxyCertDirInjectable = getInjectable({
```

Comment:
This could be wonderfully unit-testable either locally, or preferably as part of a larger system ;)